evm/spec/tables/keccak-sponge.tex

```diff
@@ -1,4 +1,76 @@
-\subsection{Keccak sponge}
+\subsection{KeccakSponge}
 \label{keccak-sponge}
 
-This table computes the Keccak256 hash, a sponge-based hash built on top of the Keccak-f[1600] permutation.
+This table computes the Keccak256 hash, a sponge-based hash built on top of the Keccak-f[1600] permutation. An instance of KeccakSponge takes as input a Memory address $a$,
+a length $l$, and computes the Keccak256 digest of the memory segment starting at $a$ and of size $l$. An instance can span many rows, each individual row being a single call to
+the Keccak table. Note that all the read elements must be bytes; the proof will be unverifiable if this is not the case. Following the Keccak specifications, the input string is padded to the next multiple of 136 bytes. 
+
+In the current implementation, the constant values are:
+\begin{itemize}
+    \item \texttt{KECCAK\_RATE\_BYTES} = 136
+    \item \texttt{KECCAK\_RATE\_U32S} = 34
+    \item \texttt{KECCAK\_CAPACITY\_U32S} = 16
+    \item \texttt{KECCAK\_DIGEST\_BYTES} = 32
+    \item \texttt{KECCAK\_WIDTH\_MINUS\_DIGEST\_U32S} = 42
+\end{itemize}
```

Comment:
Not sure this is super meaningful? That's just the specs of Keccak, and the length conversion from say u32 to byte is pretty straightforward on its own